### ISSUE TITLE
HelmRelease OCIRepository

### ISSUE THREAD

**Issue by thekoma** [tarioch/pre-commit-hooks#10]
Looks like OCIRepository  as a format per HelmRelease is not recognised:

Example:

```
apiVersion: source.toolkit.fluxcd.io/v1beta2
kind: OCIRepository
metadata:
  name: vault-secrets-webhook
  namespace: flux-system
spec:
  interval: 5m0s
  url: oci://ghcr.io/bank-vaults/helm-charts/vault-secrets-webhook
  ref:
    semver: ">= 1.*.*"
```

```
apiVersion: helm.toolkit.fluxcd.io/v2
kind: HelmRelease
metadata:
  name: vault-secrets-webhook
  namespace: vault
spec:
  interval: 10m
  chartRef:
    kind: OCIRepository
    name: vault-secrets-webhook
    namespace: flux-system
  values:
    replicaCount: 1
    debug: true
    configMapMutation: true
    secretsMutation: true
    configMapFailurePolicy: Fail
    podsFailurePolicy: Fail
    secretsFailurePolicy: Fail
  ref:
    semver: ">= 1.*.*"

```

**Comment by tarioch** [comment]
There was #8 and #9 which should have helped with this, any idea why that is not working?

**Comment by tarioch** [comment]
I hope this is fixed with #11
